jsonnet/kube-prometheus/addons/anti-affinity.libsonnet

```diff
@@ -18,7 +18,7 @@
     },
   },
 
-  local antiaffinity(labelSelector, namespace, type, topologyKey) = {
+  antiaffinity(labelSelector, namespace, type, topologyKey): {
```

Comment:
Should we move it to `lib/`?

Reply:
I wonder if it would be meaningful since this function will most likely not be used anywhere else. Theoretically it should go there, but I don't think it is meaningful to use this function outside the scope of the anti-affinity addon. The only use-case I see is to add components that are not in kube-prometheus, but in that case you would most likely also want to include the whole addon.

Reply:
Makes sense :+1: